test/e2e/ems/ems_test.go

```diff
@@ -96,3 +96,30 @@ func TestElasticMapsServerVersionUpgradeToLatest7x(t *testing.T) {
 
 	test.RunMutations(t, []test.Builder{esWithLicense, ems}, []test.Builder{esWithLicense, emsUpgraded})
 }
+
+func TestElasticMapsServerVersionUpgradeToLatest8x(t *testing.T) {
+	// todo skipping until https://github.com/elastic/cloud-on-k8s/issues/5251 is fixed.
+	t.SkipNow()
+	srcVersion := test.Ctx().ElasticStackVersion
+	dstVersion := test.LatestVersion8x
+
+	test.SkipInvalidUpgrade(t, srcVersion, dstVersion)
+
+	name := "test-ems-version-upgrade-8x"
+	es := elasticsearch.NewBuilder(name).
+		WithESMasterDataNodes(3, elasticsearch.DefaultResources).
+		WithVersion(srcVersion)
```

Comment:
Given that you are keeping Elasticsearch on `srcVersion` here I don't think we need to skip this test.